executor_test.go

```diff
@@ -2987,7 +3003,16 @@ func TestExecutor_Execute_SetRow(t *testing.T) {
 }
 
 func benchmarkExistence(nn bool, b *testing.B) {
-	c := test.MustRunCluster(b, 1)
+	c := test.MustNewCluster(b, 1)
+	var err error
+	c[0].Config.DataDir, err = ioutil.TempDir(*TempDir, "benchmarkExistence")
+	if err != nil {
+		b.Fatalf("getting temp dir: %v", err)
+	}
+	err = c.Start()
+	if err != nil {
+		b.Fatalf("starting cluster: %v", err)
+	}
```

Comment:
I didn't want to change any interfaces in our `test` package, so I'm setting up the data dir manually after newing the cluster instead of adding a parameter that we can pass in to `MustRunCluster` which will tell it where to put files.